### DISCUSSION TITLE
Error on Python Chain Ladder documentation pages

### DISCUSSION THREAD

**Discussion by Obalus2022** [casact/chainladder-python#319] [General]
Dont know if anyone here maintains, but this looks broken.

![image](https://user-images.githubusercontent.com/110035421/181866458-afa5cb65-36d5-4a65-a82d-90542362b274.png)

**Comment by kennethshsu** [comment]
Thanks for flagging this. 

We have been having trouble knowing why this is happening, because the [source code](https://github.com/casact/chainladder-python/blob/master/docs/tutorials/deterministic-tutorial.ipynb) is clean and compiled correctly. But either way, I'll look into this more.

**Comment by kennethshsu** [comment]
This is now resolved.
